engine/src/mac-qt-recorder.mm

```diff
@@ -808,6 +796,31 @@ static bool available_compression_ids(void *context, unsigned int p_id, const ch
     return true;
 }
 
+struct FormatTable
+{
+    const char *label;
+    OSType value;
+};
+
+static FormatTable record_formats[] =
+{
+    { "aiff", kQTFileTypeAIFF },
+    { "wave", kQTFileTypeWave },
+    { "ulaw", kQTFileTypeMuLaw },
+    { "movie", kQTFileTypeMovie },
+};
```

Comment:
When you have a table of constants like this that shouldn't ever get modified at runtime, it can make a significant difference to define them `static const` rather than just `static`!

- It lets the compiler make more aggressive optimisations, because it can assume the contents will never change -- in some cases, it means the array can never actually appear in the compiled code at all
- When you have a non-`const` static array, it has to get copied into the process image at startup, which requires more time and RAM. If it's `const`, it's used directly from the mapped binary

In this particular instance it probably doesn't make that much difference, though. :)